pkg/tsdb/cloudwatch/models/settings.go

```diff
@@ -12,6 +12,7 @@ type CloudWatchSettings struct {
 	awsds.AWSDatasourceSettings
 	Namespace               string `json:"customMetricsNamespaces"`
 	SecureSocksProxyEnabled bool   `json:"enableSecureSocksProxy"` // this can be removed when https://github.com/grafana/grafana/issues/39089 is implemented
+	LogsTimeout             string `json:"logsTimeout"`
```

Comment:
Right here I think the type of LogsTimeout can already be time.Duration. I think I would move the validation of the json and the initialization to default logsTimeout here too.

this bit:
```golang
    logsTimeoutDuration := e.logsTimeoutDefault

	if logsTimeout != "" {
		logsTimeoutDuration, err = time.ParseDuration(logsTimeout)
	}

	if err != nil {
		return nil, err
	}
```